tests/flow/test_function_calls.py

```diff
@@ -987,4 +987,66 @@ def test35_min_max(self):
         query = "UNWIND [1, 2, '1' ,'2' ,[1] ,[2] ,1 ,2, '1', '2', NULL, True] AS x RETURN max(x), min(x)"
         actual_result = graph.query(query)
         expected_result = [[2, [1]]]
-        self.env.assertEquals(actual_result.result_set, expected_result)
\ No newline at end of file
+        self.env.assertEquals(actual_result.result_set, expected_result)
+
+
+    def test36_log(self):
```

Comment:
please test calling `log` and `log10`  on invalid values e.g. `RETURN log10(True)`